### ISSUE TITLE
EfficientNet V2 

### ISSUE THREAD

**Issue by alancarlosml** [lukemelas/EfficientNet-PyTorch#309]
Hi @lukemelas,

Did you finish the upgrade of the code to version 2 of EfficientNet?

Thank you!

**Comment by SharanSMenon** [comment]
This project hasn't been active since Apr 2021.

**Comment by abhuse** [comment]
If anyone still interested, I have implemented V2 with pretrained weights. I converted the weights provided by original authors from tensorflow to pytorch format and achieved similar accuracy as original TF implementation without any finetuning/retraining.
https://github.com/abhuse/pytorch-efficientnet
